### PR TITLE
GTK 3.20 :: Optimized notebook padding 2

### PR DESCRIPTION
**Pull request by megax**
**Before:**
image

**After:**
image

### CODE REVIEW

**Comment by megax** [comment]
@khurshid-alam Optimized padding. Again... :) (@lots0logs somehow removed git :D)

File: src/gtk-3.20/scss/widgets/_notebook.scss [hunk 1]
```diff
@@ -155,7 +155,7 @@
                 > box,
                 > label,
                 > widget > box {
-                    padding: ($spacing + 2) ($spacing * 3); // top/bottom | left/right
+                    padding: ($spacing - 1px) ($spacing * 2); // top/bottom | left/right
```
Comment (khurshid-alam) [inline src/gtk-3.20/scss/widgets/_notebook.scss:158]: @megax 

```
padding: $spacing ($spacing * 2); 
```

better no? It's symmetrical.

File: src/gtk-3.20/scss/widgets/_notebook.scss [hunk 2]
```diff
@@ -166,7 +166,7 @@
                     > box,
                     > label,
                     > widget > box {
-                        padding: ($spacing + 2) ($spacing * 2) ($spacing + 2) ($spacing * 3); // top | right | bottom | left
+                        padding: ($spacing - 1px) ($spacing * 2) ($spacing - 1px) ($spacing * 3); // top | right | bottom | left
```
Comment (khurshid-alam) [inline src/gtk-3.20/scss/widgets/_notebook.scss:169]: @megax 

Again why this non-symmetrical padding? left-right should be 2*top-bottom. Can you test with

`padding: $spacing ($spacing * 3);`

Does it break the theme?

Comment (megax) [reply src/gtk-3.20/scss/widgets/_notebook.scss:169]: @lots0logs by asymmetrically.

File: src/gtk-3.20/scss/widgets/_notebook.scss (hunk 1 above)
Comment (megax) [reply src/gtk-3.20/scss/widgets/_notebook.scss:158]: rightful =/

**Comment by megax** [comment]
@khurshid-alam Corrected and tested.

**Comment by lots0logs** [comment]
@khurshid-alam @megax I really wish this wouldnt have been merged so quickly. I want to discuss why you guys keep making everything smaller and smaller with less and less padding. Tabs should be larger than the text that is inside of them. This does not look good at all :unamused:

**Comment by lots0logs** [comment]
> @khurshid-alam Optimized padding. Again... :) (@lots0logs somehow removed git :D)

btw, I did it on purpose.

**Comment by khurshid-alam** [comment]
Top/bottom padding is bit too large in my view as it's >toolbar height and == headerbar height (toolbar & headerbar were reduced recently). So it suffers on anything less than 1920x1080. It's hard to find a middle ground but 5/10 or 6/12 should be good for both.

**Comment by lots0logs** [comment]
> (toolbar & headerbar were reduced recently).

A change that I was also against but I digress....

The tabs in the "After" screenshot don't have enough padding in this case. It might as well just be the text without any container at all (IJS).

**Comment by khurshid-alam** [comment]
Yes. That screenshot is using  4/8 which is bit too low. 5/10 is what we are using in 3.18.  

And I didn't realize that megax changed and added back removed code.

**Comment by khurshid-alam** [comment]
Adwaita Compact one of the prime example of symmetric padding (https://github.com/abihf/adwaita-compact). It can grow from very minimal to very large.

And now that Gsk is merged with Gtk-3.24/4.0 something like this will also be possible in near future: Scalable-Adwaita
https://github.com/GNOME/gtk/tree/wip/lapo/scalable-adwaita

**Pull request merged**